### PR TITLE
Fixes empty identifier issue

### PR DESCRIPTION
**Pull request by gbrunton**
Fixes this issue:
https://stackoverflow.com/a/66070044/384853

Now you can have empty opening and closing identifiers.

### CODE REVIEW

File: QueryBuilder/Compilers/Compiler.cs [hunk 1]
```diff
@@ -834,6 +834,8 @@ public virtual string WrapValue(string value)
             var opening = this.OpeningIdentifier;
             var closing = this.ClosingIdentifier;
 
+            if (string.IsNullOrWhiteSpace(opening) && string.IsNullOrWhiteSpace(closing)) return value;
```
Comment (ahmad-moussawi) [inline QueryBuilder/Compilers/Compiler.cs:837]: Doesn't this fail when passing an empty string for the opening only?

Comment (Askolein) [reply QueryBuilder/Compilers/Compiler.cs:837]: Previous behaviour would not test for inconsistent `opening ` and  `closing` values.
Looks like this fix preserves that behaviour.

**Comment by m4ss1m0g** [comment]
Any news on when this PR can be merged? I have the same problem.

**Comment by iulian0512** [comment]
bump

**Comment by caroncyp** [comment]
Any news on this issue ?

**Comment by ahmad-moussawi** [comment]
Merged now

**Pull request merged**